src/test/java/net/rakugakibox/springbootext/logback/access/test/AccessEventAssert.java

```diff
@@ -140,6 +140,20 @@ public S hasRemoteAddr(String remoteAddr) {
     }
 
     /**
+     * Verifies that the remote address is equal to the given one.
+     *
+     * @param localPort the remote address.
+     * @return this instance.
+     */
+    public S hasLocalPort(int localPort) {
+        int actualLocalPort = actual.getLocalPort();
+        Assertions.assertThat(actualLocalPort)
+                .isNotEqualTo(NA)
```

Comment:
Can you change the parameter to `SENTINEL`?
`getLocalPort()` returns `SENTINEL`.

https://github.com/akihyro/spring-boot-ext-logback-access/pull/14/files#diff-4ae93438483ccfa280689788bd60af41R84